### ISSUE TITLE
Bug(migrate): No files match the pattern

### ISSUE THREAD

**Issue by saeidi-dev** [jsverse/transloco#795]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Which Transloco package(s) are the source of the bug?

Transloco

### Is this a regression?

Yes

### Current behavior

I am trying to migrate from ngx-translate to transloco, but I get the error No files match the pattern?!

```
$ ng g @jsverse/transloco:migrate
? ✍ Which library did you use? NgxTranslate
? ✍ Which folder will contain the translation files? ./src/app

Starting migration script

Migrating HTML files 📜
✔ Step 1/2: Migrating directives                                                                                                                                                                          
⚠️ No files match the pattern: C:\Users\Saeidi\source\frontend-v3\src\app\**\*.html
✔ Step 2/2: Migrating pipes
⚠️ No files match the pattern: C:\Users\Saeidi\source\frontend-v3\src\app\**\*.html
⚠️ No files match the pattern: C:\Users\Saeidi\source\frontend-v3\src\app\**\*.html

Migrating TS files 📜
✔ Step 1/5: Migrating modules
⚠️ No files match the pattern: C:\Users\Saeidi\source\frontend-v3\src\app\**\*.ts
⚠️ No files match the pattern: C:\Users\Saeidi\source\frontend-v3\src\app\**\*.ts
⚠️ No files match the pattern: C:\Users\Saeidi\source\frontend-v3\src\app\**\*.ts
✔ Step 2/5: Migrating service imports
⚠️ No files match the pattern: C:\Users\Saeidi\source\frontend-v3\src\app\**\*.ts
⚠️ No files match the pattern: C:\Users\Saeidi\source\frontend-v3\src\app\**\*.ts
⚠️ No files match the pattern: C:\Users\Saeidi\source\frontend-v3\src\app\**\*.ts
✔ Step 3/5: Migrating constructor injections
⚠️ No files match the pattern: C:\Users\Saeidi\source\frontend-v3\src\app\**\*.ts
✔ Step 4/5: Migrating service usage
⚠️ No files match the pattern: C:\Users\Saeidi\source\frontend-v3\src\app\**\*.ts
✔ Step 5/5: Migrating specs
⚠️ No files match the pattern: C:\Users\Saeidi\source\frontend-v3\src\app\**\*spec.ts

              🌵 Done! 🌵
Welcome to a better translation experience 🌐

For more information about this script please visit 👉 https://jsverse.github.io/transloco/docs/migration/ngx

Nothing to be done.
``` 

### Expected behavior

Migration and changes in the project files are done according to the documentation.

### Please provide a link to a minimal reproduction of the bug, if you won't provide a link the issue won't be handled.

https://codesandbox.io/s/ngneat-transloco-kn52hs

### Please provide the environment you discovered this bug in

```markdown
Transloco: 7.4.3
Angular:  17.4.1
Node: 22.5.1
Package Manager: NPM 10.8.2
OS: Windows 11
```

### I would like to make a pull request for this bug

No

**Comment by mrmemar** [comment]
I have the same problem

**Comment by aminhavasi** [comment]
yes. I have the same problem

**Comment by shaharkazaz** [comment]
@saeidi-dev Duplicate of #700 or is it something else?

**Comment by shaharkazaz** [comment]
Duplicate of #700.
Fixed in v7.4.4
